cmd/ocagent/exparser/exparser.go

```diff
@@ -12,19 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package exporter
+// This package provides support for parsing and creating the
+// respective exporters given a YAML configuration payload.
+// For now it currently only provides statically imported OpenCensus
+// exporters like:
+//  * Stackdriver Tracing and Monitoring
+//  * DataDog
+//  * Zipkin
+
+package exparser
```

Comment:
nit: `exparser` sounds confusing, may name this package to `exporterparser` or just `exporter`?

Reply:
Yeah I also wondered about that name but:
* exporterparser is too long for a Go package name
* exporter clashes with the already available package names

I'll just bite the bullet and go with `exporterparser` for clarity and that can be changed later on since anyways it is an internal package to `cmd/ocagent/internal/`

Reply:
SGTM

Reply:
I've updated it, PTAL.